app/views/user/AllCoursesCertificatesView.coffee

```diff
@@ -0,0 +1,191 @@
+require('app/styles/user/certificates-view.sass')
+RootView = require 'views/core/RootView'
+User = require 'models/User'
+Classroom = require 'models/Classroom'
+Course = require 'models/Course'
+CourseInstance = require 'models/CourseInstance'
+Campaign = require 'models/Campaign'
+LevelSessions = require 'collections/LevelSessions'
+Levels = require 'collections/Levels'
+ThangTypeConstants = require 'lib/ThangTypeConstants'
+ThangType = require 'models/ThangType'
+utils = require 'core/utils'
+fetchJson = require 'core/api/fetch-json'
+locale = require 'locale/locale'
+CourseInstances = require 'collections/CourseInstances'
+
+
+module.exports = class AllCoursesCertificatesView extends RootView
```

Comment:
why not extend this class from CertificatesView instead of copying that code?

Reply:
@mrfinch that was my initial approach, but at the end there were so many differences that I thought it's better to have them separated. I'm not sure though :-/

Reply:
They don't look very different at the end apart from a few functions but this is not blocking.